File: 01_Language/01_Functions/java/src/main/java/com/onevgo/functions/IsInfinite.java
package com.onevgo.functions;

public class IsInfinite {
    public static void main(String[] args) {
        System.out.println(Double.isInfinite(Math.sqrt(2)));
        System.out.println(Double.isInfinite(Math.log(0)));
        System.out.println(Double.isInfinite(Math.asin(2)));
    }
}

File: 01_Language/01_Functions/php/is_infinite.php
<?php

echo sqrt(2)."\n";
var_dump(is_infinite(sqrt(2))); // false

echo log(0)."\n"; // -INF
var_dump(is_infinite(log(0))); // true

echo asin(2)."\n"; // NAN
var_dump(is_infinite(asin(2))); // false

var_dump(is_infinite(null)); // false
